lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -146,6 +151,7 @@ def store(credit_card, options = {})
         add_network_transaction_reference(post, options)
         options[:recurring_contract_type] ||= 'RECURRING'
         add_recurring_contract(post, options)
+        add_recurring_detail_reference(post, options)
```

Comment:
I'm thinking this is the same as the unstore method where this has to be recurringDetailReference. I can't find the documentation but it's the same API

Reply:
Done!